_posts/2022-11-10-Improving-our-json-performance.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title:  Improving on our json Lua module
+excerpt: We've had a long standing request to implement a `json.encode()` function in our `json` Lua module.
```

Comment:
I think there is some limitations to what characters you are allowed to use here in the front matter. Did it accept the : in the https link?